.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,152 @@
+name: Build and upload to PyPI
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    tags:
+      - "*"
+  release:
+    types:
+      - published
+
+concurrency:
+  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
+  cancel-in-progress: true
+
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+
+jobs:
+  build_wheels:
+    if: "github.repository == 'MDAnalysis/mdaencore'"
+    name: Build wheels
+    runs-on: ${{ matrix.buildplat[0] }}
+    timeout-minutes: 15
+    strategy:
+      fail-fast: false
+      matrix:
+        buildplat:
+          - [ubuntu-20.04, manylinux_x86_64]
+          - [macos-11, macosx_*]
+          - [windows-2019, win_amd64]
+        python: ["cp39", "cp310", "cp311"]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.11.2
+        env:
+          CIBW_BUILD: ${{ matrix.python }}-${{ matrix.buildplat[1] }}
+          CIBW_BUILD_VERBOSITY: 1
+
+      - name: upload artifacts
+        if: |
+          (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')) ||
+          (github.event_name == 'release' && github.event.action == 'published')
+        uses: actions/upload-artifact@v2
+        with:
+          path: wheelhouse/*.whl
+          retention-days: 7
+
+  build_sdist:
+    if: "github.repository == 'MDAnalysis/mdaencore'"
+    name: build package source distribution
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build sdist
+        run: pipx run build --sdist
+
+      - name: upload artifacts
```

Comment:
From this section, do we get the wheels in PRs where tags are pushed?

Reply:
What do you mean by "in PRs"?

When you push tags the wheels will be uploaded to testpypi directly so you can pull them down and test as necessary. You'll also get the artifacts stored locally but that's the exact same thing as whatt's on testpypi (unless the testpypi upload failed).

Reply:
Oh, oops, that's why you're removing the pull trigger.